packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -93,16 +97,32 @@ export class Relayer {
               relayableDeposit,
               reason: relayStatus.relayDisputable.reason,
             });
-            // TODO: Optionally dispute the relay here if DISPUTE_MODE=true
+            // If Disputer feature is enabled, submit a dispute on-chain. Otherwise skip to the next deposit.
+            if (this.botModes.disputerEnabled) {
```

Comment:
We're only considering _existing deposits_ in this loop, right? What happens if someone relays a completely fabricated deposit. For instance, a totally invalid chainId or a deposit Id that is way past the current deposit Id?

Reply:
> We're only considering existing deposits in this loop, right?

Yep IIUC. The outermost loop considers all `relayableDeposits` which are constructed from the list of `FundsDeposited` events emitted by the `BridgeDepositBox`.

> What happens if someone relays a completely fabricated deposit
If the deposit parameters were not emitted from a `BridgeDepositBox` then we won't consider them in this loop.

And, if someone relays incorrectly a _valid_ deposit, then this PR will either skip or dispute the relay.

Reply:
Isn't this a problem? If someone fabricates a non-existent deposit, it wouldn't be disputed by this bot?